projects/1.16.1/assets/water-strainer/waterstrainer/lang/zh_cn.json

```
@@ -0,0 +1,24 @@
+{
+	"itemGroup.waterstrainer.main": "Water Strainer",
+	
+	"block.waterstrainer.strainer_base": "滤网基座",
+	"block.waterstrainer.worm_bin": "生饵桶",
+	
+	"item.waterstrainer.efficiency_meter": "效率计",
+	"item.waterstrainer.garden_trowel": "园林铲",
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
	"item.waterstrainer.garden_trowel": "园艺铲",
```